spacetemplate/repository_blackbox_test.go

```diff
@@ -0,0 +1,230 @@
+package spacetemplate_test
+
+import (
+	"testing"
+
+	"golang.org/x/net/context"
+
+	"github.com/almighty/almighty-core/gormsupport/cleaner"
+	"github.com/almighty/almighty-core/gormtestsupport"
+	"github.com/almighty/almighty-core/migration"
+	"github.com/almighty/almighty-core/spacetemplate"
+
+	"github.com/almighty/almighty-core/resource"
+
+	uuid "github.com/satori/go.uuid"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+type TestRepositorySuite struct {
+	gormtestsupport.DBTestSuite
+	clean             func()
+	spaceTemplateRepo spacetemplate.Repository
+	ctx               context.Context
+}
+
+func TestRepository(t *testing.T) {
+	t.Parallel()
+	resource.Require(t, resource.Database)
+	suite.Run(t, &TestRepositorySuite{DBTestSuite: gormtestsupport.NewDBTestSuite("../config.yaml")})
+}
+
+func (s *TestRepositorySuite) SetupTest() {
+	s.clean = cleaner.DeleteCreatedEntities(s.DB)
+}
+
+func (s *TestRepositorySuite) TearDownTest() {
+	s.clean()
+}
+
+func (s *TestRepositorySuite) SetupSuite() {
+	s.DBTestSuite.SetupSuite()
+	//s.clean = cleaner.DeleteCreatedEntities(s.DB)
```

Comment:
I have to do some more testing because somehow my space templates pollute the database and don't clean up properly. Currently I delete after each test and not when the whole suites ends.